views/components/ConfigurationCard/index.jsx

```diff
@@ -10,14 +10,13 @@ import {
   IconButton,
   FormControlLabel,
   Tooltip,
-} from '@material-ui/core'
-import { Edit, Clear, Share } from '@material-ui/icons'
-import FullView from '@material-ui/icons/AspectRatio'
-import Copy from '@material-ui/icons/FileCopy'
+} from '@mui/material'
+import { Edit, Clear, Share } from '@mui/icons-material'
+import FullView from '@mui/icons-material/AspectRatio'
+import Copy from '@mui/icons-material/FileCopy'
```

Comment:
should we make this 1 import statement by adding the second 2 along with the first import?

Reply:
[6b310d1](https://github.com/AMP-SCZ/dpdash/pull/458/commits/6b310d18b25129974d8ce470610170164ed5d426)